deployment/ansible/group_vars/staging

```diff
@@ -17,6 +17,9 @@ redis_bind_address: "{{ hostvars['database']['ansible_ec2_local_ipv4'] }}"
 csrf_session_key: "{{ lookup('env', 'DRIVER_CSRF_KEY') | default('seekrit', true) }}"
 cookie_secret_key: "{{ lookup('env', 'DRIVER_COOKIE_KEY') | default('seekrit', true) }}"
 
+## CELERY SETTINGS
+celery_host_address: "{{ hostvars['celery']['ansible_ec2_local_ipv4'] }}"
```

Comment:
I think this should work. You might be able to test locally by changing https://github.com/WorldBank-Transport/DRIVER/blob/develop/Vagrantfile#L23 to `staging:children` and reprovisioning, but I'm not sure if that'll really be representative of what the staging environment actually looks like.